atom/index.test.ts

```diff
@@ -317,4 +328,34 @@ test('prevents notifying when new value is referentially equal to old one', () =
   clock.runAll()
 })
 
+test('notifies the subscribed listener with current and old values for a store that had an initial value', () => {
```

Comment:
Can you resolve conflict (I also changed branch to `next` where we work on the next version).

Without it I can’t run CI.